src/language_server_api.h

```diff
@@ -1178,6 +1178,24 @@ MAKE_REFLECT_STRUCT(lsSignatureHelp,
                     signatures,
                     activeSignature,
                     activeParameter);
+
+// MarkedString can be used to render human readable text. It is either a markdown string
+// or a code-block that provides a language and a code snippet. The language identifier
+// is sematically equal to the optional language identifier in fenced code blocks in GitHub
+// issues. See https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting
+//
+// The pair of a language and a value is an equivalent to markdown:
+// ```${language}
+// ${value}
+// ```
+//
+// Note that markdown strings will be sanitized - that means html will be escaped.
+struct lsMarkedString {
+  std::string language;
+  std::string value;
+};
```

Comment:
this is deprecated in favour of [MarkupContent](https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#new-markupcontent), but that is currently unsupported in lsp-mode, and would require a lot more work.

Reply:
I wonder if this is why vscode is not recognizing the hover?